src/Containers/Microsoft.NET.Build.Containers/Resources/Strings.Designer.cs

```diff
@@ -447,6 +447,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to CONTAINER2006: Unknown AppCommandInstruction &apos;{0}&apos;. Valid instructions are {1}..
+        /// </summary>
+        internal static string UnknownAppCommandInstruction {
```

Comment:
I've added this manually again.

Maybe it only works on Windows?

@baronfel when you run the top-level build script on Windows, does it update this file with new strings?